h/static/styles/annotation.scss

```diff
@@ -60,39 +90,61 @@ $annotation-card-left-padding: 10px;
   margin-left: 5px;
 }
 
+.annotation-body {
+  @include font-normal;
+  color: $grey-6;
+  // Align the baseline of the text with the bottom of a containing block element.
```

Comment:
Again here I think it would be better for this comment to state the spacing that we are trying to achieve.